client/src/pages/Register/index.jsx

```diff
@@ -117,6 +118,7 @@ const SignupForms = () => {
 					errors={formErrors}
 					values={secondStepValues}
 					setValues={setSecondStepValues}
+					setAddress={setAddress}
```

Comment:
you can use the setValues instead of passing setAddress and not add new state (as mentioned above)